ui-participant/src/Navbar.tsx

```diff
@@ -52,6 +53,8 @@ export default function Navbar(props: NavbarProps) {
   const navLinks = localContent.navbarItems
 
   async function updatePreferredLanguage(selectedLanguage: string) {
+    //track the language change
+    mixpanel.track('languageUpdated', { language: selectedLanguage, source: 'navbar' })
```

Comment:
this seemed to be weird for me - if I changed from en -> es it didn't fire one, then es -> dev fired one for the es language, then dev -> es fired one for dev. I wonder if there's some weird component refresh thing going on? Perhaps it needs to be in a `useEffect`?

Reply:
I haven't been able to reproduce this :( I have seen related behavior in the past though, where selecting a language was similar out of date by one click. Have never been able to intentionally reproduce it though